src/components/announcement-dialog/AnnouncementDialog.tsx

```diff
@@ -72,16 +59,24 @@ export const AnnouncementDialog = forwardRef<
         }}
         open={open || !announcementSeen}
       >
-        <StyledImage $imageWidth={imageWidth}>{image}</StyledImage>
-        <Content spacing={8}>
+        <div
+          className={styles.styledImage}
+          style={{
+            '--image-margin': imageWidth ? '0 auto' : 'unset',
+            '--image-width': imageWidth ? `${imageWidth}px` : '100%',
+          }}
+        >
```

Comment:
I think we might want to namespace the css variable here. As it maybe conflict other component that's using the same css 🤔 Maybe we need to discuss more about it.